src/lib/inbox/inbox.ts

```diff
@@ -90,7 +90,13 @@ export class InboxTools {
 
     // we take a long average block time of 14s
     // and always move at least 10 blocks
-    const diffBlocks = Math.max(Math.ceil(diff / this.l1Network.blockTime), 10)
+    let diffBlocks = 10
+    if ((this.l1Network as L1Network).blockTime) {
```

Comment:
`L2Network` does not have `blockTime`. I'm not sure if we need to add `blockTime` to every L2 network that is a parent chain to L3?

Reply:
see https://github.com/OffchainLabs/arbitrum-sdk/pull/319 for an attempted fix

Reply:
I didn't include this fix because it's out of scope of this PR and the bridge UI doesn't require.
I can create a separate PR to add this.